webapp/channels/src/components/user_account_menu/user_account_profile_menuitem.tsx

```diff
@@ -51,6 +43,18 @@ export default function UserAccountProfileMenuItem(props: Props) {
         }]));
     }
 
+    function handleClick() {
+        dispatch(openModal({
+            modalId: ModalIdentifiers.USER_SETTINGS,
+            dialogType: UserSettingsModal,
+            dialogProps: {isContentProductSettings: false},
+        }));
+
+        if (isCompleteYourProfileTaskPending) {
```

Comment:
Is this fixing a bug from the previous PR?

Reply:
oh yes it does